app.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python2.7
+
+"""
+Amity Room Allocation System
+Usage:
+  create_room <room_type> <room_name>
+  add_person <first_name> <second_name> <email> <FELLOW|STAFF> [wants_accommodation]
+  reallocate_person <email> <new_room_name>
+  load_people
+  print_allocations [-o=filename] 
+  print_unallocated [-o=filename]
+  print_room <room_name>
+  save_state [--db=sqlite_database] 
+  load_state <sqlite_database>
+Options:
+    -h, --help  Show this screen and exit
+"""
+
+import cmd
+import shutil
+import click
+from docopt import docopt, DocoptExit
```

Comment:
Why are you using both click and docopt? Choose one and settle for it as it will give you the same functionality both ways.